packages/vkui/src/components/PanelHeaderButton/PanelHeaderButton.tsx

```diff
@@ -59,7 +59,7 @@ export const PanelHeaderButton = ({
 
   switch (platform) {
     case Platform.IOS:
-      hoverMode = 'background';
+      hoverMode = '';
```

Comment:
На айпадах нет никакой отдачи от кнопок при наведении мышкой

```suggestion
      hoverMode = 'opacity';
```